tests/helpers/bitcoin.ts

```diff
@@ -1,10 +1,17 @@
 /* tslint:disable:no-relative-imports */
 const Client = require('bitcoin-core')
-import { delay } from '../helpers/utils'
+import {LoggingConfiguration, Configuration, loadConfigurationWithDefaults} from 'Configuration'
+import { createModuleLogger } from 'Helpers/Logging'
+import * as Pino from 'pino'
+import {delay, delayInSeconds} from '../helpers/utils'
 
 const BITCOIND_A = 'bitcoind-1'
 const BITCOIND_B = 'bitcoind-2'
 
+// @ts-ignore
```

Comment:
can you add a comment shortly describing the need for this?

Reply:
oops... I think it's accidental... removing